src/components/FormHeading/index.js

```diff
@@ -4,9 +4,21 @@ import "./FormHeading.css";
 
 const FormHeading = ({ text }) => {
   return (
-    <Typography variant="h4" align="left" className="FormHeading_heading">
-      {text}
-    </Typography>
+    <>
+      <Typography
+        variant="h4"
+        align="left"
+        className="FormHeading_heading FormHeading_BigScreen"
+      >
+        {text}
+      </Typography>
+      <Typography
+        variant="body2"
+        className="FormHeading_heading FormHeading_PhoneScreen"
+      >
+        {text}
+      </Typography>
+    </>
```

Comment:
Are you hiding/showing based on screen size here? Why not leverage the responsive font size feature of typography? 
https://github.com/tc39/proposal-nullish-coalescing#overview-and-motivation